Buie landing page and new menu (#612)
box/developer.box.com: content/pages/box-ui-elements/index.md

---
hide_title: true
alias_paths: []
centered: false
---
# Box AI Developer Zone

<Centered wide id="buie" >
  <HeroImage type="BUIE" imageWidth="548" imageHeight="493">
    <Header>
      Create</br>
      experiences with</br>
      Box UI Elements
    </Header>

> Box UI Elements are pre-built UI components that allow extending custom portals with the elements of the main Box web application.

Reviewer: ```suggestion
Box UI Elements are pre-built UI components that allow you to extend custom portals with the elements of the main Box web application.
```

  </HeroImage>
</Centered>

<Centered mid>
  <Header>
    Interactive Demos
  </Header>
  <p style="text-align: left; margin-left: 0;">
    Box UI Elements are available both as React components and framework-agnostic JavaScript libraries.
  </p>
  <BuieDemo></BuieDemo>

  <More to='/guides/embed/ui-elements/installation/' center>
    Get started
  </More>
</Centered>

<Centered mid>
  <Header>
    Explore more Box UI Elements
  </Header>
  <p style="text-align: left; margin-left: 0;">

>     Follow along with additional guides and documentation 

Reviewer: ```suggestion
    Follow more guides and documentation 
```

    to help get you up and running.
  </p>

  <TileGrid rows="4">
    <Tile type="ai" title="Box AI for UI Elements"
      href="/guides/embed/ui-elements/preview/#box-ai-for-ui-elements">

>         See how to embed Box AI in Content Preview UI Element.

Reviewer: ```suggestion
        See how to embed Box AI in the Content Preview UI Element.
```

    </Tile>
    <Tile type="upload" title="Content Uploader"
      href="/guides/embed/ui-elements/uploader/">
        Embed this component and allow users to upload files.
    </Tile>
    <Tile type="ui-metadata" title="Metadata view"
      href="/guides/embed/ui-elements/explorer/#metadata-view">

>         Display files based on specified metadata with Content Explorer.

Reviewer: ```suggestion
        Display files based on specified metadata with the Content Explorer.
```

    </Tile>
    <Tile type="annotations" title="Annotations"
      href="/guides/embed/ui-elements/annotations/">
        Provide collaboration capabilities within the Content Preview.
    </Tile>
  </TileGrid>
</Centered>

<Centered mid>
  <Header centered>
    Build custom portals
  </Header>
  Learn more from videos, blog posts and sample code related to Box UI Elements.

  <TileGrid rows="3">
    <Tile image="BUIE-portal" title="Read more about creating custom portals"
      href="https://medium.com/box-developer-blog/build-a-content-portal-using-box-ui-elements-react-tailwind-css-vercel-part-1-f1c509621ceb">

>         Build a content portal with customized experience using Box UI Elements.

Reviewer: ```suggestion
        Build a content portal with a customized experience using Box UI Elements.
```
